syne_tune/optimizer/schedulers/searchers/gp_fifo_searcher.py

```diff
@@ -449,6 +454,36 @@ class GPFIFOSearcher(ModelBasedSearcher):
         because criterion is only used internally. Also note that criterion
         data is always normalized to mean 0, variance 1 before fitted with a
         GP.
+    transfer_learning_task_attr : str (optional)
+        Used to support transfer HPO, where the state contains observed data
+        from several tasks, one of which is the active one. To this end,
+        `configspace` must contain a categorical parameter of name
+        `transfer_learning_task_attr`, whose range are all task IDs. Also,
+        `transfer_learning_active_task` must denote the active task, and
+        `transfer_learning_active_config_space` is used as
+        `active_config_space` argument in :class:`HyperparameterRanges` (this
+        allows us to use a narrower search space for the active task than for
+        the union of all tasks (`configspace` must be that).
+    transfer_learning_active_task : str (optional)
```

Comment:
```
        allows us to use a narrower search space for the active task than for
        the union of all tasks (`configspace` must be that).
```
Can we say in one sentence on how this is done?

Reply:
I've added something. The reationale here is that the ranges in active_config_space may be smaller than what is needed to cover all the configs over all tasks. For example, say you want a HP to lie in a certain range, but you have some data where the range was larger. This allows you to use that old data.